Peer/GUI/guimanager.h

```diff
@@ -0,0 +1,40 @@
+#ifndef GUIMANAGER_H
+#define GUIMANAGER_H
+
+#include <QObject>
+#include "GUI/frienditem.h"
+#include "GUI/friendmodel.h"
+
+#include "dalmanager.h"
+
+class GUIManager : public QObject
+{
+  Q_OBJECT
+
+  Q_PROPERTY(QObject* friendModel READ friendModel NOTIFY friendModelChanged)
+
+public:
+  explicit GUIManager(QObject *parent = nullptr);
+
+  FriendModel* friendModel();
```

Comment:
You should delete default copy constructor and operator == or implement deep copying

Reply:
I think we will not copy GUIManager. Am I right?